### ISSUE TITLE
Adding a mailbox without configuring "Fetching Emails"

### ISSUE THREAD

**Issue by PhillerEU** [freescout-helpdesk/freescout#2563]
We want to add a mailbox to our freescout but only with "Sending email" configured, not "Fetching emails" 
We have an incoming mailbox that will sort out and distribute the emails using workflows. I am not able to choose the mailbox in which we haven't configured "Fetching emails", because of that reason i guess. 
Is there any workaround/solution where we can have a working mailbox i can choose in workflows, but doesn't have the Fetching emails" configured? 

The reason we want to do this: We experience that our freescout suddenly stops fetching all mailboxes, perhaps because we have so many (22). We want to solve this issue by doing the above mentioned: One incoming mailbox that needs to fetch, sort and move to specific mailbox, and 21 others that only needs to be able to send emails. 
We are currently running 1.8.47.

**Comment by freescout-helpdesk** [comment]
Unfortunately not possible for now. 

Have you tried to diagnose the fetching issue? https://github.com/freescout-helpdesk/freescout/wiki/Fetching-Emails#troubleshooting-fetching-issues

Are you using docker, Cloudron?

**Comment by PhillerEU** [comment]
We have tried to diagnose the issue but with no real progress. We can fetch all mailboxes but they seem to be losing connection at random intervals. 
Normally it runs just fine but every couple of days it spits out a connection error on all active mailboxes at once: 

Error: connection setup failed; File: /var/www/html/vendor/webklex/php-imap/src/Client.php (391) 

This can happen up to a few times a day. 
Our fix, to this day, is to just manually fetch twice and then it works fine. Obviously it seems to be a connection issue and we believe it's because of the fact that we fetch from so many IMAPs.

**Comment by freescout-helpdesk** [comment]
Yes, this must be a limitation from the mail service side on a number of connections per minute/hour.

**Comment by freescout-helpdesk** [comment]
Have you tried to use `flock` utility: https://github.com/freescout-helpdesk/freescout/wiki/Fetching-Emails#common-issues

**Comment by freescout-helpdesk** [comment]
In the `master` branch we've added a small delay between connections.

You can try this updated "FetchEmail.php" file and play with the value on this line: https://github.com/freescout-helpdesk/freescout/blob/master/app/Console/Commands/FetchEmails.php#L101

**Comment by petervansark** [comment]
Had the same issue since yesterday. Using IMAP and M$.
Can confirm this fixed the issue.
Thank you!
